package/utils/util-linux/Makefile

```diff
@@ -172,7 +172,7 @@ endef
 define Package/fdisk
 $(call Package/util-linux/Default)
   TITLE:=manipulate disk partition table
-  DEPENDS:= +libblkid +libsmartcols +libfdisk
+  DEPENDS:= +libblkid +libsmartcols +libfdisk +libncursesw
```

Comment:
Given that `fdisk` is rarely (if ever used), it's not clear that it needs wide-byte support or NLS. How much smaller is it with NLS turned off?  Because libncursesw.so is about 8.4MB long...

But I imagine there are a lot of other things already pulling it in, so maybe it's a moot point.